src/t8_cmesh/t8_cmesh.c

```diff
@@ -1159,8 +1159,10 @@ t8_cmesh_reset (t8_cmesh_t *pcmesh)
     sc_MPI_Comm comm;
     /* Check whether a correct communicator was stored at tree_offsets.
      * This is useful for debugging. */
-    comm = t8_shmem_array_get_comm (cmesh->tree_offsets);
-    T8_ASSERT (t8_cmesh_comm_is_valid (cmesh, comm));
+    if (t8_cmesh_is_committed (cmesh)) {
+      comm = t8_shmem_array_get_comm (cmesh->tree_offsets);
+      T8_ASSERT (t8_cmesh_comm_is_valid (cmesh, comm));
+    }
```

Comment:
No need to review these lines, they are already in https://github.com/DLR-AMR/t8code/pull/869

Reply:
This is PR #869 , do you mean #870 ?